pom.xml

```diff
@@ -76,6 +76,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
make the built executable jar to be called: OnlineServicesEmulator
the original jar should be called: OnlineServicesEmulator-1.0-SNAPSHOT